haystack/components/evaluators/context_relevance.py

```diff
@@ -67,6 +67,7 @@ class ContextRelevanceEvaluator(LLMEvaluator):
     def __init__(
         self,
         examples: Optional[List[Dict[str, Any]]] = None,
+        progress_bar: bool = True,
```

Comment:
Docstring.

Reply:
thanks, anyway the linter was supposed to catch this - I have to check what's going on